MidCamp/midcamp
# 48 Event Page

Completes https://trello.com/c/jXC6uyNv/48-event-page

## To test:
- Create events (with depth)
- Create news, sponsors, venue
- Visit the parent event page, observe views for Sub-events, news, sponsors, venue
